parsl/executors/ipp.py

```diff
@@ -112,26 +112,21 @@ def start(self):
         self.launch_cmd = command_composer(self.engine_file, self.engine_dir, self.container_image)
         self.engines = []
 
-        if self.provider:
```

Comment:
This was left in to allow for users to potentially set provider=None, but no one would use that anymore.

Reply:
Ah, thanks for the clarification!